src/openforms/js/lang/en.json

```diff
@@ -399,6 +394,11 @@
     "description": "Logic type selection label",
     "originalDefault": "Please select the logic type:"
   },
+  "9LdZVP": {
+    "defaultMessage": "Attribute",
+    "description": "Variable prefill attribute label",
+    "originalDefault": "Attribute"
+  },
```

Comment:
I assume the actual translations will be added in a separate PR?

Reply:
English translations are pretty much untouched - we try to get the source strings/defaults right the first time :grimacing:

Reply:
ah I didn't see this was `en.json`, makes sense then :p